cli/commands/version/version.go

```diff
@@ -0,0 +1,61 @@
+package version
+
+import (
+	"runtime"
+	"text/template"
+
+	"github.com/monitoror/monitoror/cli"
+	"github.com/monitoror/monitoror/cli/version"
+	"github.com/monitoror/monitoror/pkg/templates"
+
+	"github.com/spf13/cobra"
+)
+
+const versionTemplate = ` Version:    {{green .Version}}{{ with .BuildTags }}{{printf " (%s)" . | grey }}{{end}}
+ Git commit: {{green .GitCommit}}
+ Built:      {{green .BuildTime}}
+
+ Go version: {{blue .GoVersion}}
```

Comment:
Useless?

Reply:
`monitoror version`  command print all build information
`monitoror --version` build only monitoror version

It's two different needs (like docker, kubernetes, and many other does)